core/src/main/java/org/dromara/dynamictp/core/DtpRegistry.java

```diff
@@ -44,11 +45,7 @@
 import org.springframework.boot.ApplicationArguments;
 import org.springframework.boot.ApplicationRunner;
 
-import java.util.Collections;
-import java.util.List;
-import java.util.Map;
-import java.util.Objects;
-import java.util.Set;
+import java.util.*;
```

Comment:
Please replace with a single class import and we can use IDEA checkstyle plugins to check it (the file in .github/checkstyle/checkstyle.xml)